docs/development/i18n.rst

```diff
@@ -2,23 +2,65 @@ Internationalization (i18n)
 ===========================
 
 The code, templates and JavaScript user visible strings must all be
-wrapped with `gettext`_ functions to be substituted at runtime with
-the equivalent localized string. These function names are used as
-markers to collect the strings to be translated and store them into
-the ``securedrop/translations/messages.pot`` file. For each language
+wrapped with `gettext`_ functions to be substituted with the
+equivalent localized string. For instance:
+
+.. code:: python
+
+     if not (msg or fh):
+       flash(gettext("You must enter a message or choose a file to submit."), "error")
+       return redirect(url_for('main.lookup'))
+
+The ``gettext`` function name is used as
```

Comment:
This sounds like it's *only* used as a marker. Maybe say "extractors look for calls to the function `gettext` when collecting strings for translation."

Reply:
done